lesson15/index.js

```diff
@@ -6,6 +6,7 @@ const showModalBtn = document.getElementById("js-btn-showModal");
 const modal = document.getElementById("js-modal");
```

Comment:
Why do you get main element by tag name? (line1)

Sorry, I couldn't comment line1 directly.
So, I commented here.

Reply:
Thanks for the review!
I hadn't noticed that it was id'd but not used.
I fixed it to use the id.

483090d